### ISSUE TITLE
1 km FIT Count data entry: date picker obscured

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/PoMS#82]
The current date picker opens up but is partially hidden behind the map.

**Comment by kitenetter** [comment]
See what effect the new design has.

**Comment by andrewvanbreda** [comment]
AVB to test for May 2021 release once new design available

**Comment by andrewvanbreda** [comment]
Still occurring on latest theme.

**Comment by andrewvanbreda** [comment]
@CCarvell 
I have a fix I can deploy for this one if you don't mind site being in maintenance mode for 5 minutes.

**Comment by andrewvanbreda** [comment]
@CCarvell 
Note: I tried to deploy this this evening, but there was a problem. I had originally pushed map to the back so the date picker isn't obscured, but I put this onto live but noticed last minute this breaks map clicking.
Fortunately I noticed and reversed this before taking out of maintenance mode.
Will have to push date picker to front instead.
Site was down for about 10 minutes with no alteration made in the end.
